chromadb/db/index/hnswlib.py

```diff
@@ -28,6 +29,8 @@
     "hnsw:resize_factor": r"^\d+(\.\d+)?$",
 }
 
+# Trigger hooks
```

Comment:
Is this anything special? Or just a no-op commit to trigger the hooks?

Reply:
Was an accident - removed!